Koans/AboutConstants.cs

```diff
@@ -0,0 +1,114 @@
+using DotNetCoreKoans.Engine;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+    public class AboutConstants : Koan
+    {
+        const string C = "nested";
```

Comment:
I do not think this example is relevant to C#. The ruby koans demonstrates that constants can be declared globally or in a class definition. Global constants are not really a thing in C#.